### ISSUE TITLE
AutoType doesn't seem to inherit correctly

### ISSUE THREAD

**Issue by GuardianMajor** [keepassxreboot/keepassxc#8339]
## Overview
[TIP]:  # ( DO NOT include screenshots of your actual database! )
[NOTE]: # ( Give a BRIEF summary about your problem )

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. set these options
![image](https://user-images.githubusercontent.com/5274181/183277126-0096e87a-873e-48ae-b4cd-440297a7a865.png)
![image](https://user-images.githubusercontent.com/5274181/183277168-d6f7daa5-bf46-416d-9e3f-79788e7d7169.png)
3. create a group
4. check the same settings that are inherited
![image](https://user-images.githubusercontent.com/5274181/183277226-d1204ebe-6209-4f72-bb86-c24f120157b2.png)
![image](https://user-images.githubusercontent.com/5274181/183277231-724965e6-0a7c-4a95-acb2-2974da40b503.png)

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
All inherited options should be the same as the parent group right?

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )
As you can see, they are all good with the exception of the auto-type.

## Context
[NOTE]: # ( Give us any additional information you may have. )
I have the portable version if it matters, don't think it does but just in case.

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - 2.7.1
KeePassXC-Browser - 1.8.1
Operating system: Win64
Browser: Microsoft Edge 103.0.1264.77

[NOTE]: # ( Pick choices based on your environment )
Operating System: Windows

**Comment by droidmonkey** [comment]
Is that from the NEW group page? If so, check it again when editing that same group. This is just a visual problem because the inheritance is tested for functionality in our unit tests.

**Comment by GuardianMajor** [comment]
Yes, that's on the new group page, I didn't even think of it being a visual problem, let me check to confirm that.

You are absolutely correct, it was a visual because once I clicked ok (while it was showing incorrectly like above) and then went back into "Edit Group", it was showing correctly.

Thank you.

Edit. Apologies for closing it, I realized after I did that, you might want it open to handle the new label.

**Comment by GuardianMajor** [comment]
A follow up question to make sure I am seeing this correctly. If the group has this setting:

![image](https://user-images.githubusercontent.com/5274181/183332995-5e67f085-f60c-418b-8a69-f7019cd595bb.png)

and an entry is created within it, shouldn't this option be off?

![image](https://user-images.githubusercontent.com/5274181/183333066-a2d32762-fee7-4e59-a3cb-4f8b17fcb493.png)

**Comment by droidmonkey** [comment]
Yes, but just another visual clue. We did that with the browser settings and it made sense, just need to do the same for auto-type.

**Comment by GuardianMajor** [comment]
Understood, thank you. Let me know if I can help with anything.
